frontend/src/components/AccountPicker.vue

```diff
@@ -5,23 +5,32 @@ import { Network, networkName, useEthereumStore } from '../stores/ethereum';
 import JazzIcon from './JazzIcon.vue';
 import { abbrAddr } from '@/utils/utils';
 import { useMedia } from '@/utils/useMediaQuery';
+import { MetaMaskNotInstalledError } from '@/utils/errors';
 
 const eth = useEthereumStore();
 
 const netName = computed(() => networkName(eth.network));
 const unkNet = computed(() => eth.network === Network.Unknown);
 
 const connecting = ref(false);
-const showingConnecting = ref(false);
+const isMetaMaskInstalled = ref(true);
 
 async function connectWallet() {
+  if (!isMetaMaskInstalled.value) {
```

Comment:
Can you also fix the mouse pointer resembling something clickable over the connect wallet button.

![image](https://github.com/oasisprotocol/demo-voting/assets/83672/c87efa21-1590-441e-b3f7-2586ffbcdade)